webargs/core.py

```diff
@@ -477,6 +484,9 @@ def location_handler(self, name):
 
             @parser.location_handler('name')
             def parse_data(request, name, field):
+                # Marshmallow 3
```

Comment:
Drive-by comment: `request` here is a hypothetical Request object, not a `MarshalResult`, so no need to change the docs here

Reply:
Oh, right. Fixed. (I'll squash all commits when done.)